.azure-pipelines/ut.yml

```diff
@@ -79,3 +79,57 @@ jobs:
         export PATH=/usr/local/mpi/bin:$PATH
         mpirun -tag-output -x MSCCLPP_HOME=$(System.DefaultWorkingDirectory) -np 8 python3 -m pytest ./python/test/test_mscclpp.py -x
       workingDirectory: '$(System.DefaultWorkingDirectory)'
+
+  - task: Bash@3
```

Comment:
Use another pipeline job for this？ It will start a new container and don't need to clean previous binaries

Reply:
Fixed.